wrappers/javascript/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -172,8 +172,8 @@ export class NodeJSAnoncreds implements Anoncreds {
       attributeRawValues as unknown as Buffer,
       attributeEncodedValues as unknown as Buffer,
       revocationRegistryId,
-      revocationStatusList,
-      revocationConfiguration.ref(),
+      revocationStatusList ?? 0,
+      revocationConfiguration ? revocationConfiguration.ref().address() : 0,
```

Comment:
Same for here. I think just the value / undefined works.

Reply:
In this case it's at compilation time where I get the `number | undefined is not assignable to number`. Maybe there is a way for refType to allow it?